spec/integration/data_stream/manifest.spec.yml

```diff
@@ -52,6 +52,7 @@ spec:
               - email
               - integer
               - password
+              - select
```

Comment:
What do current versions of kibana do when finding this type?

Reply:
They would render a text field (empty in this case), which is the fallback in case the type is unknown.

Reply:
Ok, I guess this is fine then, users can still configure the packages.